app/services/google_calendar/busy_time_ranges_for_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module GoogleCalendar
+  class BusyTimeRangesForUser < ApplicationService
+    def initialize(user:, time_range:)
+      @user = user
+      @time_range = time_range
+    end
+
+    def call
+      # TODO, implement calendar integration
```

Comment:
To be implemented in a separate PR. App is not live so I don't mind creating an empty service I can stub out in the tests. Same for Propositions::Create.